WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -345,6 +347,11 @@ class MySiteViewModel
         _onDynamicCardMenuShown.postValue(Event(model))
     }
 
+    private fun onQuickStartTaskTypeItemClick(type: QuickStartTaskType) {
+        quickStartRepository.clearActiveTask()
```

Comment:
1. Question (❓): Why is `quickStartRepository.clearActiveTask()` needed, what does it do and what will happen if we don't call this clear function?
2. Question (❓): There is another `clearActiveQuickStartTask()` view model function here, which is utilised during fragment's `onPause()` under certain conditions. Should that be reused to avoid duplication and feature branching later on? Does this suppose to serve the same use case?

Reply:
1. This logic is copied from the [list item click action logic from the `MySiteFragment`](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt#L716
) where it clears the active quick start task. I'd like to retain the logic here.

2. [`MySiteFragment` clears the active task `onPause`](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt#L383) as well, so yes `clearActiveQuickStartTask` can be re-utilized, it does the same thing. I didn't see it existed, thank you for noticing it 🙏. - replaced in 985e314

Reply:
👋 @ashiagr !

1. On my word, I really did navigation to the [list item click action logic from the `MySiteFragment`](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt#L716
) but for some reason didn't noticed that, thanks so much for noticing it and retaining the login here as well. 🙏 🌟 
2. Great, thanks so much! 🙇